.github/actions/frontend/runtime/e2e_run/action.yml

```diff
@@ -148,8 +153,8 @@ runs:
         CYPRESS_ENVIRONMENT_NAME: jupiterone-dev
         GITHUB_TOKEN: ${{ inputs.github_token }}
         MIGRATION_NUMBER: ${{ inputs.migration_number }}
-        PR_NUMBER: ${{ inputs.commit_info_pr_number || github.event.pull_request.number }}
```

Comment:
This was the primary issue as github.event.pull_request.number doesn't exists in the context of the issue_comment event. So the test run was referencing an invalid magic url.